extensions/amp-sticky-ad/0.1/amp-sticky-ad.css

```diff
@@ -29,17 +29,21 @@ amp-sticky-ad {
 
 .-amp-sticky-ad-layout {
   display: flex;
+  visibility: hidden;
   flex-direction: column;
   align-items: center;
   justify-content: center;
   overflow: visible !important;
+  transform: translateZ(0) !important;
 }
 
 .amp-sticky-ad-loaded {
   background-color: #fff;
 }
 
 .-amp-sticky-ad-layout > amp-ad {
+  /* need a background-color other than default to prevent blink */
+  background-color: rgba(255,255,255,0);
```

Comment:
you're right. I didn't see blink with multiple tests. I guess `transform` did the work. I will remove it for now.